src/input/input.cpp

```diff
@@ -635,7 +635,9 @@ static const int mapToIndex[] =
     0,
     16, 17, 18, 19, 20,
     0, 0, 0, 0, 0, 0, 0, 0,
-    21, 22, 23, 24, 25
+    21, 22, 23, 24, 25,
+    0, 0, 0, 0, 0, 0, 0,
+    26, 27, 28,
```

Comment:
The indexes of these three values need to match the D, E and F constants in `input.h`.

The `BUTTON_CODE_COUNT` constant above need to have the highest number here +1.